draft_1_proposal_instructions.md

```diff
@@ -0,0 +1,144 @@
+In the future, the contents of the "SNS lifecycle" section in ./README.md would
```

Comment:
Is there a reason to rephrase parts (e.g., steps 0 and 1) that are not affected by the new 1-proposal design?